decidim-system/spec/system/organizations_spec.rb

```diff
@@ -88,6 +88,29 @@
       end
     end
 
+    describe "edits organization name" do
+      let!(:organization) do
+        create(:organization, name: { ca: "", en: "Citizen Corp", es: "" }, default_locale: :en, available_locales: ["en"], description: { en: "large text" })
```

Comment:
You're creating an organization that doesn't do anything. I understand that's done because you were having an ambiguous match as you had two flash messages. You can fix that error by adding a `match: :first` in line 108 (i.e. `within ".flash__message", match: :first do`)